reana_client/cli/workflow.py

```diff
@@ -888,7 +896,7 @@ def workflow_validate(ctx, file):  # noqa: D301
     for p in ctx.params:
         logging.debug("{param}: {value}".format(param=p, value=ctx.params[p]))
     try:
-        load_reana_spec(click.format_filename(file))
+        load_reana_spec(click.format_filename(file), environments)
```

Comment:
The validation is working inside this `load_reana_spec()`. 

I can avoid it but I will duplicating the code for reading the `yaml` file and loading the specs if it is okay?

Reply:
... or just make the new argument optional, so that we don't have to amend existing `load_reana_spec()` callers? Kind of like `skip_validaton_of_environments=True`? A bit like `skip_validation=False` is used... So if you add a new option at the end, and it will be optional, the branch could still go into `maint-0.7` I think.  We can expand this later more nicely to cover all the various validation scenarios that are to be skipped...